numpyro/distributions/transforms.py

```diff
@@ -1417,7 +1415,7 @@ def _inverse(self, y: jnp.ndarray) -> jnp.ndarray:
         return y
 
     def extend_axis_rev(self, array: jnp.ndarray, axis: int) -> jnp.ndarray:
-        normalized_axis = normalize_axis_tuple(axis, array.ndim)[0]
+        normalized_axis = axis if axis >= 0 else jnp.ndim(array) + axis
```

Comment:
what about this change?

Reply:
This raises a warning that this function is hidden and accessing it through numpy.core.numeric is deprecated.